base/applications/cmdutils/whoami/lang/it-IT.rc

```diff
@@ -36,22 +36,21 @@ BEGIN
     IDS_UNKNOWN_DESCRIPTION "???"
     IDS_STATE_ENABLED "Abilitato"
     IDS_STATE_DISABLED "Disabilitato"
-    IDS_ERROR_UPN "ERRORE: Impossibile trovare il nome dell'utente principale (UPN) perchè l'utente attualmente registrato\nnon è un dominio utente.\n"
-    IDS_ERROR_FQDN "ERRORE: Impossibile trovare il nome distinto pienamente qualificato (FQDN) perchè l'utente\nattualmente registrato non è un dominio utente.\n"
-    IDS_ERROR_VALUEXPECTED "ERRORE: Sintassi non valida. È previsto un valore per '/fo'.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
-    IDS_ERROR_VALUENOTALLOWED "ERRORE: Sintassi non valida. '%s' valore non autorizzato per l'opzione '/fo'.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
-    IDS_ERROR_1TIMES "ERRORE: Sintassi non valida. L'uso dell'opzione '%s' non è autorizzato più\n di una volta.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
-    IDS_ERROR_INVALIDSYNTAX "ERRORE: Sintassi non valida.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
-    IDS_ERROR_INVALIDARG "ERRORE: Argomento o opzione non valida - '%s'.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
-    IDS_ERROR_NH_LIST "ERRORE: L'opzione /NH non può essere usata in formato LIST.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
+    IDS_ERROR_UPN "ERRORE: Impossibile trovare l'User Principal Name (UPN) perchè l'utente attualmente registrato\nnon è un dominio utente.\n"
+    IDS_ERROR_FQDN "ERRORE: Impossibile trovare il Fully Qualified Domain Name (FQDN) perchè l'utente\nattualmente registrato non è un dominio utente.\n"
```

Comment:
Maybe "[...] un Fully Qualified Domain Name (FQDN) perchè [...]" is better?

Reply:
I'm on it.